x-pack/plugins/security_solution/server/lib/timeline/routes/README.md

```diff
@@ -323,4 +323,292 @@ kbn-version: 8.0.0
      "timelineId":"f5a4bd10-83cd-11ea-bf78-0547a65f1281", // This is a must as well
      "version":"Wzg2LDFd" // Please provide the existing timeline version
 }
-```
\ No newline at end of file
+```
+
+## Export timeline api
+
+#### POST /api/timeline/_export
+
+##### Authorization
+
+Type: Basic Auth
+
+username: Your Kibana username
+
+password: Your Kibana password
+
+
+
+
+##### Request header
+
+```
+
+Content-Type: application/json
+
+kbn-version: 8.0.0
+
+```
+
+##### Request param
+file_name	${filename}.ndjson
```

Comment:
nit, but can you add a colon `:` in between these? its hard to read in the rich view

Reply:
add also wrap it in a code block, it gets lost otherwise